pygsp/graphs/_io.py

```diff
@@ -373,7 +373,7 @@ def from_graphtool(cls, graph, weight='weight'):
         >>> v = graph.add_vertex()
         >>> eprop = graph.new_edge_property("double")
         >>> eprop[e1] = 0.2
-        >>> eprop[(1, 2)] = 0.9
```

Comment:
this is not working anymore with graph_tool

Reply:
Good catch thanks!